01/components/Table.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import TableHeader from './TableHeader';
+import TableBody from './TableBody';
+import TableFooter from './TableFooter';
+
+class Table extends React.Component {
+    render() {
+        const {data} = this.props;
+
+        return (
+            <section className='Table'>
+                <TableHeader /> 
```

Comment:
Nagłówek też można by było zrobić uniwersalny tj. jakie dane nie są przekazywane to on generuje odpowiednie nagłówki z odpowiednią liczbą kolumn :)